features/area-types.authorised.feature

```diff
@@ -0,0 +1,96 @@
+Feature: Area Types Private Endpoints Enabled
```

Comment:
is there a convention for a line between every step?

Reply:
HelloWorld.feature didn't have this

Reply:
I think there isn't but with a discussion with Rhys (@redhug1 ) we mentioned that we should have 1 line between declarations, and 2 lines between scenarios.
Also, the body or information about a step should be indented with an extra level, so that it can be collapsed in IDEs or editors.